src/api/registrationApi.ts

```diff
@@ -81,12 +82,30 @@ export const createDraftDoi = async (registrationId: string) =>
     method: 'POST',
   });
 
+const userIsAuthenticated = async () => {
+  try {
+    const cognitoUser = await Auth.currentAuthenticatedUser();
+    if (cognitoUser) {
+      return true;
+    } else {
+      return false;
+    }
+  } catch {
+    return false;
+  }
+};
```

Comment:
dette føles veldig bloated, og jeg hater det. Må se om jeg finner en penere løsningen. Dette henger litt sammen med bumping av aws-amplify, som vi har en egen PR på også: https://github.com/BIBSYSDEV/NVA-Frontend/pull/4947

Reply:
Flytter det over til `authApi.ts`, og lar det være sånn. Blir en liten performance hit for alle oppslag på resultat, men tror ikke det skal være merkbart